config/locales/de.yml

```diff
@@ -920,7 +920,7 @@ de:
     order_state:
       address: Adresse
       awaiting_return: erwartet Erstattung
-      canceled: Abgebrochen
+      canceled: Storniert
```

Comment:
I was more concerned about the key than the translation, but wanted to check it wasn't a find/replace mistake